ddtrace/internal/opentelemetry/context.py

```diff
@@ -79,7 +83,6 @@ def detach(self, token):
 
     @property
     def _ddcontext_provider(self):
-        # type: () -> DDBaseContextProvider
```

Comment:
nit: Are we removing these other type hints because it's all or none? Or consistency?

Reply:
I went for the all or none approach. I don't have a strong opionon here. I guess something is better than nothing. Also these are python2 compatible type hints. We need to remove them and replace them with python3 type hints anyway.